assemble/bin/start-daemon.sh

```diff
@@ -103,6 +103,9 @@ if [[ "$SERVICE" != "tserver" || $NUM_TSERVERS -eq 1 ]]; then
    fi
    echo "Starting $SERVICE on $HOST"
 
+   OUTFILE="${ACCUMULO_LOG_DIR}/${SERVICE}_1_${LOGHOST}.out"
```

Comment:
If you don't want the _1 for just one tserver, we could remove the changes to the top section.